outputs.tf

```diff
@@ -14,6 +14,16 @@
  * limitations under the License.
  */
 
-output "bucket_name" {
-  value = google_storage_bucket.main.name
+output "instance" {
+  value       = google_data_fusion_instance.instance
+  description = "The created CDF instance"
+}
+
+locals {
+  tenant_project_re = "cloud-datafusion-management-sa@([\\w-]+).iam.gserviceaccount.com"
+}
+
+output "tenant_project" {
```

Comment:
we should also output service endpoint, to make it easy to use this module and cdap provider.

Reply:
Note this currently does not address #4

Reply:
A user can do `module.cdf_instance.instance.service_endpoint` since the entire instance is returned. Is that ok?

Reply:
oh yeah! that's fine!